packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -344,6 +344,8 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
       callback,
       isReplace: false,
       isForced: false,
+      isCapture: false,
+      capturedValue: null,
```

Comment:
This sounds generic and not obviously error related. Do you have other uses?

(Replying to myself: blockers you dummy.)